### PR TITLE
initial use cases docs

### PR DESCRIPTION
**Pull request by laugustyniak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# TREF - Text Representation Evaluation Framework
+
+State-of-the-art Text Representations for Natural Language Processing tasks, an initial version of library focus on the Polish Language
+
+# Installation
```
Comment (ktagowski) [inline README.md:5]: why not installation from pypi?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# TREF - Text Representation Evaluation Framework
```
Comment (ktagowski) [inline README.md:1]: What is the origin of this name :D?

File: README.md (hunk 1 above)
Comment (laugustyniak) [reply README.md:5]: I forgot ;p however we must check and build pypi package quite often. I got `ModuleNotFoundError: No module named 'typer'` after `pip install embeddings` command.

Comment (ktagowski) [reply README.md:5]: but our package name is different try `pip install clarinpl-embeddings`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+State-of-the-art Text Representations for Natural Language Processing tasks, an initial version of library focus on the Polish Language
+
+# Installation
+
+```bash
+pip install embeddings
```
Comment (ktagowski) [inline README.md:8]:
```suggestion
pip install clarinpl-embeddings
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+# Run example tasks
+
+```bash
+cd .\examples
+```
+
+## Run classification task
+
+```python
```
Comment (ktagowski) [inline README.md:19]:
```suggestion
```bash
```

File: README.md (hunk 1 above)
Comment (ktagowski) [reply README.md:5]: Anyway we have already automatic pypi package building

Comment (laugustyniak) [reply README.md:5]: done

File: README.md (hunk 2 above)
Comment (laugustyniak) [reply README.md:1]: it is just an idea, wdyt? do you have any propositions?

Comment (ktagowski) [reply README.md:1]: I think it should be same as our package name to not confuse users. And therefore all path should be changed e.g. `embeddings` module should fit our destination name e.g. `tref`

**Review by ktagowski — APPROVED** [review]

**Pull request merged**